config/quizServicies.php

```diff
@@ -193,6 +210,7 @@
 foreach ($container->findTaggedServiceIds('controller') as $id => $tags) {
     $controller = $container->getDefinition($id);
     $controller->addMethodCall('setSession', [$session]);
+    $controller->addMethodCall('setCodeHighLighter', [$codeHighLighter]);
```

Comment:
Every controller uses the highlighter lib/session? I don't think so. Inject it where needed.